xyz-util/src/test/java/com/here/xyz/test/util/geo/GeometryValidatorIT.java

```diff
@@ -45,4 +48,56 @@ public void testInvalidGeometry() throws InvalidGeometryException {
         Assertions.assertThrows(GeometryException.class, () -> GeometryValidator.validateGeometry(point, 0));
     }
 
+    @Test
+    public void testIsWorldBoundingBox_withValidBBoxes() {
+        PolygonCoordinates expectedCoordinates = new PolygonCoordinates();
+
+        // Expected coordinates of a world bounding box
+        LinearRingCoordinates lrc = new LinearRingCoordinates();
+        lrc.add(new Position(-180.0, -90.0));
+        lrc.add(new Position(180.0, -90.0));
+        lrc.add(new Position(180.0, 90.0));
+        lrc.add(new Position(-180.0, 90.0));
+        lrc.add(new Position(-180.0, -90.0));
+
+        expectedCoordinates.add(lrc);
+        Polygon polygon = new Polygon().withCoordinates(expectedCoordinates);
+        Assertions.assertTrue(GeometryValidator.isWorldBoundingBox(polygon), "Should detect valid world bbox");
+
```

Comment:
These are basically two tests in one. They should be separated into 2 test methods